### PR TITLE
Add a bluebike route

### PR DESCRIPTION
**Pull request by MichielLeyman**
Adds a `GET /bluebike` route.

Assumes presence of a `bluebike.geojson` file under `/storage/app`. @pietercolpaert as discussed, you can run the scraper on cron and have it output to this location.

### CODE REVIEW

**Comment by pietercolpaert** [comment]
Should be `stations/bluebike`

File: app/Http/Controllers/BluebikeController.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return Response::make($this::getJSON())
+            ->header('Content-Type', 'application/json')
```
Comment (pietercolpaert) [inline app/Http/Controllers/BluebikeController.php]: should be application/ld+json

**Comment by pietercolpaert** [comment]
@MichielLeyman Is this ready?

**Comment by MichielLeyman** [comment]
@pietercolpaert Yes!

`/stations/bluebike` or `/stations/bluebike/{stationName}` both redirect to https://datapiloten.be/bluebike/availabilities.geojson

**Pull request merged**